File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoLayoutCardItem.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnThemeImageRenderingMode;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutCardItem extends StudioBase
{
  @Nullable
  public StudioDtoLayoutCardItemLine fifthLine;

  @Nullable
  public StudioDtoLayoutCardItemLine firstLine;

  @Nullable
  public StudioDtoLayoutCardItemLine fourthLine;

  @Nullable
  public StudioDtoColor imageBackgroundColor;

  @Nullable
  public MetaIdVar imageCornerVarId;

  @Nullable
  public Long imageHeight;

  @Nullable
  public MetaIdVar imageHeightVarId;

  @Nullable
  public EnumDefnThemeImageRenderingMode imageRenderingMode;

  @Nullable
  public Long imageWidth;

  @Nullable
  public MetaIdVar imageWidthVarId;

  @Nullable
  public MetaIdField[] mediaFieldIdSet;

  @Nullable
  public MetaIdVar[] mediaVarIdSet;

  @Nullable
  public StudioDtoLayoutCardItemLine secondLine;

  @Nullable
  public StudioDtoLayoutCardItemLine thirdLine;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutCardItem.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnThemeImageRenderingMode
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar

interface StudioDtoLayoutCardItem : StudioBase {
    val fifthLine: StudioDtoLayoutCardItemLine?
    val firstLine: StudioDtoLayoutCardItemLine?
    val fourthLine: StudioDtoLayoutCardItemLine?
    val imageBackgroundColor: StudioDtoColor?
    val imageCornerVarId: MetaIdVar?
    val imageHeight: Long?
    val imageHeightVarId: MetaIdVar?
    val imageRenderingMode: EnumDefnThemeImageRenderingMode?
    val imageWidth: Long?
    val imageWidthVarId: MetaIdVar?
    val mediaFieldIdSet: List<MetaIdField>?
    val mediaVarIdSet: List<MetaIdVar>?
    val secondLine: StudioDtoLayoutCardItemLine?
    val thirdLine: StudioDtoLayoutCardItemLine?
}
